### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by jonnetanninen**

### CODE REVIEW

File: src/modules/search/components/result/MapResults.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    const icon = L.divIcon({
+      className: 'custom-icon',
+      html: ReactDOMServer.renderToString(element),
+    });
+
+    return icon;
+  };
+
+  // Centers the map to middle of all the projects on initial render
+  const getInitialPosition = () => {
```
Comment (duubliduu) [inline src/modules/search/components/result/MapResults.tsx:71]: If you add `[number, number]` return type you can remove the `// @ts-ignore` later down.

File: src/modules/search/components/result/MapResults.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <IconLocation size={'m'} color={isActive ? 'white' : color} />
+      </div>
+    );
+
+    /*if (project.building_type === 'hitas') {
```
Comment (duubliduu) [inline src/modules/search/components/result/MapResults.tsx:59]: is this actually a comment for future implementor? and not commented code

File: src/modules/search/components/result/SearchResults.tsx [hunk 3]
```diff
@@ -3,20 +3,35 @@ import css from './SearchResults.module.scss';
...
+const SearchResults = ({ searchResults, openMap }: Props) => {
   const { t } = useTranslation();
 
   return (
     <div className={css.container}>
-      <div className={css.titleContainer}>
-        <h1>{t('SEARCH:free-apartments')}</h1>
-        <div className={css.resultsCount}>
-          {t('SEARCH:total')} {searchResults.length} {t('SEARCH:apartments')}
+      <div style={{ display: 'flex', justifyContent: 'space-between', flexWrap: 'wrap', padding: '62px 48px 32px' }}>
```
Comment (duubliduu) [inline src/modules/search/components/result/SearchResults.tsx:18]: Is there some magic going on here I don't see or why is the styles inline here...?

File: src/modules/search/components/result/SearchResults.tsx [hunk 4]
```diff
@@ -3,20 +3,35 @@ import css from './SearchResults.module.scss';
...
+        <div className={css.titleContainer}>
+          <h1>{t('SEARCH:free-apartments')}</h1>
+          <div className={css.resultsCount}>
+            {t('SEARCH:total')} {searchResults.length} {t('SEARCH:apartments')}
+          </div>
+        </div>
+        <div>
+          <Button
+            style={{ height: 56, margin: '.67em 0', display: 'flex', alignItems: 'center' }}
```
Comment (duubliduu) [inline src/modules/search/components/result/SearchResults.tsx:27]: ... here....

File: src/modules/search/components/result/SearchResults.tsx [hunk 5]
```diff
@@ -3,20 +3,35 @@ import css from './SearchResults.module.scss';
...
+            variant="secondary"
+            onClick={openMap}
+          >
+            <div style={{ display: 'flex', alignItems: 'center' }}>
```
Comment (duubliduu) [inline src/modules/search/components/result/SearchResults.tsx:31]: ... and here?

File: src/types/common.ts [hunk 6]
```diff
@@ -133,6 +133,7 @@ export type Project = {
   shareholder_meeting_date: string;
   site_area: number;
   site_renter: string;
+  state_of_sale: string;
```
Comment (duubliduu) [inline src/types/common.ts:136]: OMG, you could add the enum here :excited:

**Review by duubliduu — APPROVED** [review]
Added a couple of comments. LGTM 👍

File: src/modules/search/components/result/MapResults.tsx (hunk 1 above)
Comment (jonnetanninen) [reply src/modules/search/components/result/MapResults.tsx:71]: Tried already and tried again, does not work for some weird reason :(

File: src/modules/search/components/result/MapResults.tsx (hunk 2 above)
Comment (jonnetanninen) [reply src/modules/search/components/result/MapResults.tsx:59]: Yes

Comment (jonnetanninen) [reply src/modules/search/components/result/MapResults.tsx:59]: Actually it is some leftover, I will remove

**Pull request merged**